static/bidder-info/33across.yaml

```diff
@@ -8,5 +8,5 @@ capabilities:
     - video
 userSync:
   iframe:
-    url: "https://ic.tynt.com/r/d?m=xch&rt=html&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&ru={{.RedirectURL}}&id=zzz000000000002zzz"
+    url: "https://ssc-cms.33across.com/ps/?m=xch&rt=html&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&ru={{.RedirectURL}}&id=zzz000000000002zzz"
```

Comment:
@mike-foster can you please confirm if this is the sync url Prebid Server needs to be using? Instead of IC which per our discussions leads to lower match rate?

Reply:
Yes, that is correct. Thanks!